coverage_grcov.Makefile.toml

```diff
@@ -40,7 +40,7 @@ set -eux
 
 grcov ${COVERAGE_PROF_OUTPUT} \
   -b ${COVERAGE_BINARIES} -s ${CARGO_MAKE_WORKSPACE_WORKING_DIRECTORY} \
-  -t ${GRCOV_OUTPUT_TYPE} --llvm --branch --ignore-not-existing --ignore "/*" --ignore "*/tests/*" --ignore "*/testing/*" --ignore "target/*" --ignore "contracts/swapper/mock/*" --ignore "contracts/swapper/osmosis/*" -o ${GRCOV_OUTPUT_PATH}
+  -t ${GRCOV_OUTPUT_TYPE} --llvm --branch --ignore-not-existing --ignore "tests/osmosis_tests.rs" -o ${GRCOV_OUTPUT_PATH}
```

Comment:
why ignore osmosis tests?

Reply:
they all run against OsmosisTestApp so can't run coverage on them

Reply:
Ah. Maybe open an issue to generalize the tests to use generic test runner?